docs/Makefile

```
@@ -0,0 +1,94 @@
+	@echo "Full clean complete"
+
+# Watch for changes and rebuild automatically (requires inotifywait)
+watch:
+	@echo "Watching for changes (Ctrl+C to stop)..."
+	@while true; do \
+		inotifywait -e modify -r $(MAIN_TEX) $(PREAMBLE) $(SECTIONS_DIR)/ 2>/dev/null && \
+		$(MAKE) --no-print-directory all; \
+	done
+
+# Display help information
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (18).

(maxbodylength)